src/cson.coffee

```diff
@@ -60,6 +60,7 @@ parseContents = (objectPath, cachePath, contents, callback) ->
   try
     object = parseObject(objectPath, contents)
   catch parseError
+    parseError.message = "#{objectPath}: #{parseError.message}"
```

Comment:
> The error message comes from cson-parser and if that error message spans multiple lines (shows a code snippet at the end) then it would look weird. I know coffeescript and stylus both put the filename first.

Good point, so if you could just tack the path on the error then we can :ship: this, thanks!